raiden/network/transport/matrix/transport.py

```diff
@@ -406,10 +420,22 @@ def on_success(greenlet):
                 self.log.debug("Starting retrier", retrier=retrier)
                 retrier.start()
 
-        self.log.debug("Matrix started", config=self._config)
         super().start()  # start greenlet
+        self._starting = False
         self._started = True
 
+        self.log.debug("Matrix started", config=self._config)
+
+        # Handle any delayed invites in the future
+        self._spawn_later(self._process_queued_invites, 1)
```

Comment:
I think `_spawn_later` can easily be confused with gevents internal `spawn_later`. Maybe you could change the name slightly to something like `run_later`, `schedule_greenlet`. Having a queue for invites is a really good Idea, but there also  kinda "first class", e.g. processing them after 1 second can lead to messages being processed  and triggering join_room before the invites etc. Could we prioritize `_process_queued_invites` somehow to be processed before messages and other data which is received via sync?

Reply:
Yes, ideally we should move the whole event processing to a (priority) queue based system. But please let's do this separately. This PR is huge already.